docs/UserGuide.md

```diff
@@ -139,19 +139,23 @@ Examples:
 * `unmark 3 4 5`<br>
 Marks tasks at index 3, 4 and 5 as incomplete.
 
-### 2.8. Schedule (Work in progress) : `schedule`
+### 2.8. Schedule: `schedule`
 
 Set an event or a deadline.<br>
-Format: `schedule INDEX [STARTIME] ENDTIME`
+Format: `schedule INDEX [STARTIME to] ENDTIME`
```

Comment:
Wonder if the 'to' can be changed to something else which improves user readability & understanding. Took me a while to understand the command